src/tests/controllers/testsController.js

```diff
@@ -21,6 +22,14 @@ async function insertTestBenchMark(req, res, next) {
         return next(err);
     }
 }
+async function getBenchmark(req, res, next) {
+    try {
+        const result = await manager.getBenchmark(req.params.test_id);
```

Comment:
it will return 200/201 even if benchmark not found

Reply:
true, but this how our api work.
for example get reports: /v1/testId/reports will return empty list if not reports